### PR TITLE
rearrange the checklist into actionable items and advice sections

### PR DESCRIPTION
**Pull request by nsryan2**
## Summary of changes

This PR rearranges the checklist to have lists of actionable items and then sections for grammar, style, and advice on clarity. It also adds grammar rules for punctuation around quotations (lines 81 and 82) and about the prepositions 'of' and 'for' (line 90). Other than those specific lines, I didn't add anything new.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #
- Closes: #260 

## Associated Developers

- Dev: @

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests) to get to the 
Review the Checklist before they begin their review.

### CODE REVIEW

**Review by gwenchee — APPROVED** [review]
The new organization is great! Why do you leave [ ] for some bullets and change some bullets to - ?

**Comment by nsryan2** [comment]
The idea was that the first two sections are an actual checklist of actionable items for writers to go through. The sections about enhancing or the additional math comments are not actionable items, more so advice, or grammar rules

**Comment by nsryan2** [comment]
Maybe we could make it clearer by changing the sections headers to 
# checklist
## writing
## tables and figures

# enhancing your writing
## style
## grammar
## conciseness 
## additional math comments

**Comment by gwenchee** [comment]
Regarding this: 
> The idea was that the first two sections are an actual checklist of actionable items for writers to go through. The sections about enhancing or the additional math comments are not actionable items, more so advice, or grammar rules

It's confusing because some of the actionable checklist items do fall in the enhancing clarity and enhancing style subsections. And the way you split them up is based on how they were worded. For example `Get rid of passive voice (is/was/are/were/be/been/am + past tense verb), replace with active voice` sounds actionable, but it could be easily reworded into `avoid passive voice (is/was/are/were/be/been/am + past tense verb), and instead use active voice` and put into the enhancing clarity section. 

Having two sections: checklist and enhancing your writing might suggest to students that the checklist is mandatory, while the enhancing your writing section is a suggestion. I think the original intention of the writing checklist is that all of it is mandatory, thus a checklist to be completed. 

My suggestion is to keep the checklist and divide it into sub-sections. If you want, you could also reword some of the bullets with more actionable wording.

**Comment by nsryan2** [comment]
Great feedback! I wasn't originally thinking of it like that, but that makes more sense. I'll rework it a little bit in that framework!

File: manual/guides/writing/checklist.md [hunk 1]
```diff
@@ -10,67 +10,103 @@
...
 - [ ] Get rid of extraneous prepositions (the meeting happened on monday -> the meeting happened monday) (they agreed that it was true -> they agreed it was true)
 - [ ] Get rid of passive voice (is/was/are/were/be/been/am + past tense verb), replace with active voice
-- [ ] Use strong verbs (use sparingly: is, are, was, were, be, been, am).
-- [ ] Avoid turning verbs into nouns ("obtain estimates of" -> "estimates"; "provides a description of" -> "describes").
+- [ ] Make sure to cite all images, methods, software, and empirical data. Review [the principles](https://www.force11.org/software-citation-principles) and try [CiteAs](https://citeas.org/about) if necessary.
+```
+
+## Enhancing Clarity
+
+As Shakespeare wrote in *Hamlet*, "Brevity is the soul of wit."
```
Comment (gwenchee) [inline manual/guides/writing/checklist.md:29]: Nice

**Review by gwenchee — APPROVED** [review]
This looks great!

**Comment by nsryan2** [comment]
@gwenchee Awesome! Everything should be all good to go!

**Pull request merged**